src/test/java/org/example/features/LoginManagement.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Login Functionalities
```

Comment:
Usual feature files located in test/resources/.... 
https://cucumber.io/docs/guides/10-minute-tutorial/?lang=java#:~:text=In%20Cucumber%2C%20an%20example%20is%20called%20a%20scenario.%20Scenarios%20are%20defined%20in%20.feature%20files%2C%20which%20are%20stored%20in%20the%20src/test/resources/hellocucumber%20directory%20(or%20a%20subdirectory).

Reply:
Feuture folder moved